src/api/procedures/__tests__/createMultiSig.ts

```diff
@@ -54,17 +53,17 @@ describe('createMultiSig procedure', () => {
     const mockAccount = entityMockUtils.getAccountInstance();
 
     const signers = [mockAccount];
-    const signaturesRequired = new BigNumber(1);
+    const requiredSignatures = new BigNumber(1);
 
     const signatories = signers.map(s => utilsConversionModule.signerToSignatory(s, mockContext));
```

Comment:
(nit) rawSignatories. I think it's easier to identify at a glance which values are Codecs and which aren't if we prefix them with raw. Although to be honest I don't think `raw` is the correct word here, so you might want to think of a different prefix for the future and do a mass rename

Reply:
Btw, why aren't you using `dsMockUtils` for these?